<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckInDialog</class>
 <widget class="QDialog" name="CheckInDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QPushButton
{
	margin-bottom : 25%;
	margin-top : 25%;
	margin-left : 150px;
	margin-right : 150px;
	border-radius : 8px;
	background : rgb(127, 127, 255);
   padding : 8px;
}

.QComboBox
{
	margin-right : 20%;
}

.QLabel
{
	margin-top : 25%;
	margin-left : 50%;
	margin-right : 50%;
}

.QLineEdit
{
	margin-bottom : 25%;
	margin-left : 50%;
	margin-right : 50%;
	padding : 10px;
	border-radius : 8px;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>445</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="fioLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>ФИО</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fioEdit">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="phoneNumberLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton
{
	margin-bottom : 25%;
	margin-top : 25%;
	margin-left : 150px;
	margin-right : 150px;
	border-radius : 8px;
	background : rgb(127, 127, 255);
   padding : 8px;
}

.QLabel
{
	margin-top : 25%;
	margin-left : 50%;
	margin-right : 50%;
}

.QLineEdit
{
	margin-bottom : 25%;
	margin-left : 50%;
	margin-right : 50%;
	padding : 10px;
	border-radius : 8px;
}</string>
      </property>
      <property name="text">
       <string>Номер телефона</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneNumberEdit">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton
{
	margin-bottom : 25%;
	margin-top : 25%;
	margin-left : 150px;
	margin-right : 150px;
	border-radius : 8px;
	background : rgb(127, 127, 255);
   padding : 8px;
}

.QLabel
{
	margin-top : 25%;
	margin-left : 50%;
	margin-right : 50%;

}

.QLineEdit
{
	margin-bottom : 25%;
	margin-left : 50%;
	margin-right : 50%;
	padding : 10px;
	border-radius : 8px;
}

</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="postalIndexLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Почтовое отделение</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="postalIndexComboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="checkInButton">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
